lib/functions/function_plot_measures.R

```diff
@@ -0,0 +1,31 @@
+plot_measures <- function(
+    data,
+    title = NULL,
+    x_label = NULL,
+    y_label = NULL) {
+
+  # Create plot
+  plot <- ggplot(
+    data,
+    aes(
+      x = interval_end,
+      y = numerator,
+      color = measure,
+      group = measure
+    )
+  ) +
+    geom_line() +
+    labs(
+      title = title,
+      x = x_label,
+      y = y_label,
+      color = "Condition"
+    ) +
+    # Setting the minimum y-value
+    ylim(y_min, NA) +
```

Comment:
There are many ways to change the limits of the axes in ggplot2, with some small methodoligical differences, e.g., some methods drop the data that doesnt fall in within the limits, some methods keep the data but dont visualise it. For this reason I usually use the [scale_continuous](https://ggplot2.tidyverse.org/reference/scale_continuous.html) functions, and an additional benefit is that they come with more options to format the axis labels. Have look what you think